core/src/test/java/org/apache/iceberg/rest/requests/TestUpdateNamespacePropertiesRequest.java

```diff
@@ -45,35 +45,38 @@ public void testRoundTripSerDe() throws JsonProcessingException {
     // Full request
     String fullJson = "{\"removals\":[\"foo\",\"bar\"],\"updates\":{\"owner\":\"Hank\"}}";
     assertRoundTripSerializesEquallyFrom(
-        fullJson, UpdateNamespacePropertiesRequest.builder().updateAll(UPDATES).removeAll(REMOVALS).build());
+        fullJson, ImmutableUpdateNamespacePropertiesRequest.builder().updates(UPDATES).removals(REMOVALS).build());
```

Comment:
given that we're directly working with the `updates` / `removals` collection, it's actually easier to use directly the builder from `ImmutableUpdateNamespacePropertiesRequest` and not have simplified `of(..)` methods like in the other class